### DISCUSSION TITLE
Add hashes for all download options

### DISCUSSION THREAD

**Discussion by ikmckenz** [civitai/civitai#330] [Ideas]
Related to already implemented feature: https://github.com/civitai/civitai/discussions/98

Currently hashes are only shown for the default download of a model. If a model has multiple formats such as a pruned model or PickleTensors vs SafeTensors, hashes for those files are not available (as far as I can tell).

**Comment by WASasquatch** [comment]
Good catch.

**Comment by JustMaier** [comment]
Good point. We've got the data. It's just not being displayed in the UI.
We're overhauling the resource page and I'll see if I can find a reasonable way to display the hashes for each file on a version that doesn't overcomplicate/clutter things.

**Reply by ikmckenz** [reply]
Is this implemented now? It looks like each file in a multi-file model has a little info button with it's hashes now?
